app/models/campaign.rb

```diff
@@ -24,4 +25,44 @@ def self.current
   def show_in_sidebar?
     sidebar_enabled? && sidebar_image.present?
   end
+
+  # @return [Integer] The total number of articles in the campaign.
+  delegate :count, to: :the_scope
```

Comment:
Not a big fan of the `the_` prefix, I think I'd prefer something like `scoped_articles`.